websocket-channel-2023.html

```diff
@@ -0,0 +1,487 @@
+                <h3 property="schema:name">Conformance</h3>
+                <div datatype="rdf:HTML" property="schema:description">
+                  <p>All assertions, diagrams, examples, and notes are non-normative, as are all sections explicitly marked non-normative. Everything else is normative.</p>
+
+                  <p>The key words “MUST” and “MUST NOT” are to be interpreted as described in <cite><a href="https://tools.ietf.org/html/bcp14" rel="rdfs:seeAlso">BCP 14</a></cite> [<cite><a class="bibref" href="#bib-rfc2119">RFC2119</a></cite>] [<cite><a class="bibref" href="#bib-rfc8174">RFC8174</a></cite>] when, and only when, they appear in all capitals, as shown here.</p>
+                </div>
+              </section>
+            </div>
+          </section>
+
+          <section id="subscription-type" inlist="" rel="schema:hasPart" resource="#subscription-type">
+            <h2 property="schema:name">WebSocketChannel2023 Type</h2>
+            <div datatype="rdf:HTML" property="schema:description">
+              <p>This specification defines the <code>WebSocketChannel2023</code> type for use with Solid Notifications subscriptions that use the WebSocket Web API. The URI of the subscription type is <code>http://www.w3.org/ns/solid/notification#WebSocketChannel2023</code>.</p>
+              <p>An <code>WebSocketChannel2023</code> API MUST conform to the <cite><a href="https://solid.github.io/notifications/protocol#discovery">Solid Notifications Protocol</a></cite>. [<cite><a class="bibref" href="#bib-solid-notifications">SOLID-NOTIFICATIONS</a></cite>]</p>
+
+              <p>An <code>WebSocketChannel2023</code> API SHOULD support the <cite><a href="https://solid.github.io/notifications/protocol#notification-features">Solid Notifications Features</a></cite>.</p>
+
+              <p>A client establishes a subscription using the <code>WebSocketChannel2023</code> type by sending an authorized subscription request to the hypermedia API retrieved via <cite><a href="https://solid.github.io/notifications/protocol#discovery">Solid Notifications discovery</a></cite>.</p>
+
+              <p>For <code>WebSocketChannel2023</code> interactions, the client sends a JSON-LD payload to the appropriate hypermedia API via <code>POST</code>. The only required fields in this interaction are <code>type</code> and <code>topic</code>. <span about="" id="client-subcription-type" rel="spec:requirement" resource="#client-subscription-type"><span property="spec:statement">The <code>type</code> field <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> contain the type of subscription being requested.</span></span> <span about="" id="client-subscription-feature" rel="spec:requirement" resource="#client-subscription-feature"><span property="spec:statement">The <code>topic</code> field <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> contain the URL of the resource a client wishes to subscribe to changes.</span></span></p>
```

Comment:
Replace:
MUST</span> contain the URL of the resource a client wishes to subscribe to changes.</span></span></p>

With:
MUST</span> contain the URL of the resources a client wishes to subscribe to changes.</span></span></p>